CMakeLists.txt

```diff
@@ -107,6 +116,16 @@ endif() # TRITON_ENABLE_GPU
 #
 configure_file(src/libtriton_pytorch.ldscript libtriton_pytorch.ldscript COPYONLY)
 
+if (TRITON_PYTORCH_ENABLE_TORCHTRT)
+   set(TORCHTRT_COPY_COMMAND docker cp pytorch_backend_ptlib:/opt/conda/lib/python3.8/site-packages/torch_tensorrt/lib/libtorchtrt_runtime.so libtorchtrt_runtime.so)
```

Comment:
This block needs to be surrounded by TRITON_PYTORCH_DOCKER_BUILD. Why not move to under 129?

Reply:
Other macros set in this block are not docker-specific. I can split the block in two if that matters.

Reply:
Yes that would be ideal since we would anyway need it to function as such for when we configure it for the non-docker based Jetson build.